nms/app/packages/magmalte/app/components/admin/Alerts.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow
+ * @format
+ */
+
+import Button from '@fbcnms/ui/components/design-system/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Grid from '@material-ui/core/Grid';
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+import MenuItem from '@material-ui/core/MenuItem';
+import React, {useCallback} from 'react';
+import Select from '@material-ui/core/Select';
+import axios from 'axios';
+import useMagmaAPI from '@fbcnms/ui/magma/useMagmaAPI';
+import {makeStyles} from '@material-ui/styles';
+import {sortBy} from 'lodash';
+import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
+import {useState} from 'react';
+
+const useStyles = makeStyles(() => ({
+  header: {
+    margin: '10px',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  paper: {
+    margin: '10px',
+  },
+  noNetworks: {
+    height: '70vh',
+  },
+}));
+
+function Alerts() {
```

Comment:
I'm confused on where we would see this component? Can you share a screenshot/screen recording here? Thanks

Reply:
This is in the screen recording in the test plan.